activity-tracking/routes/goals.js

```diff
@@ -38,9 +40,9 @@ router.post('/add', async (req, res) => {
 // PUT: Update the user's goal by username (User should only have one goal)
 router.put('/update/:username', async (req, res) => {
     try {
-      const { username, exerciseType, goalType, goalUnit, goalValue } = req.body;
+      const { username, exerciseType, goalType, goalUnit, goalValue, caloriesGoal, goalAim } = req.body;
```

Comment:
What is the goal aim doing? storing the kcal aimed for right?

Reply:
goalAim is a property of the Goal class, which stores the value that the user selects as their aim, for example, fitness, weight loss etc.